generate.go

```diff
@@ -9,9 +9,8 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/Sirupsen/logrus"
+	"github.com/opencontainers/ocitools/libgen"
```

Comment:
@wking , this new package structure looks very different from the current package structure.

    ocitools
    |-- cmd
    |    `-- runtimetest
    |         |-- main.go
    |         |-- rlimit_linux.go
    |         |-- mount
    |               |-- mountinfo_freebsd.go
    |               |-- mountinfo.go
    |               |-- mountinfo_linux.go
    |               |-- mountinfo_solaris.go
    |               |-- mountinfo_unsupported.go
    |               |-- mountinfo_windows.go
    |-- main.go
    |-- generate.go
    |-- validate.go